modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -11,3 +11,35 @@ PyObject* pyopencv_from(const std::vector<GCompileArg>& value)
 {
     return pyopencv_from_generic_vec(value);
 }
+
+template <typename T>
+static PyObject* extract_proto_args(PyObject* py_args, PyObject* kw)
+{
+    using namespace cv;
+
+    GProtoArgs args;
+    Py_ssize_t size = PyTuple_Size(py_args);
+    for (int i = 0; i < size; ++i) {
+        PyObject* item = PyTuple_GetItem(py_args, i);
+        if (PyObject_TypeCheck(item, (PyTypeObject*)pyopencv_GScalar_TypePtr)) {
+            args.emplace_back(((pyopencv_GScalar_t*)item)->v);
```

Comment:
Since it is still C++, maybe we shouldn't use C-style casts here?

Reply:
Fixed